25111.1 <c-61#150> Created logger
25111.4 <c-61#150> appl_initialize was called
25111.4 <c-61#150:22> Logger was updated
25111.4 <c-61#150:22> Started intializing cube lib
25111.4 <c-61#150:22> Finished intializing cube lib with worker, starting to work
25115.1 <c-61#150:22> requestCubes => [52] Buffer size: 0
25115.1 <c-61#150:22> Sent requestCubes signal to root
25115.1 <c-61#150:22> Received 1 cubes from root
25115.1 <c-61#150:22> The main loop continues
25115.1 <c-61#150:22> Started solving a cube
25322.2 <c-61#150:22> Cube failed
25322.2 <c-61#150:22> Added failed cube
25322.2 <c-61#150:22> returnFailedCubes => [52] Buffer size: 3
25322.2 <c-61#150:22> Sent 1 failed cubes to root
25322.2 <c-61#150:22> Received receivedFailedCubes signal from root
25322.2 <c-61#150:22> requestCubes => [52] Buffer size: 0
25322.2 <c-61#150:22> Sent requestCubes signal to root
25322.2 <c-61#150:22> Received 1 cubes from root
25322.2 <c-61#150:22> The main loop continues
25322.2 <c-61#150:22> Started solving a cube
25352 <c-61#150:22> appl_interrupt was called
25352 <c-61#150:22> Interrupting worker
25352 <c-61#150:22> appl_interrupt was called
25352 <c-61#150:22> appl_withdraw was called
25352 <c-61#150:22> STOP (0.02s since last cb)
25352 <c-61#150:22> Solving interrupted
25352 <c-61#150:22> The main loop continues
25352 <c-61#150:22> Leaving the main loop due to an interruption
25352 <c-61#150:22> Started withdraw thread
25352 <c-61#150:22> Finished withdrawing lib
25352 <c-61#150:22> Enter destructor of CubeWorker
25352 <c-61#150:22> Time waiting for messages: 0.000
25352.4 <c-61#150:22> Enter destructor of CubeWorkerInterface
25352.4 <c-61#150:22> Enter destructor of CubeRoot
25352.5 <c-61#150:22> Finished withdraw thread
25413.4 <c-61#150:22> Enter destructor
25413.4 <c-61#150:22> Time working: 240.590
25413.4 <c-61#150:22> Time suspended: 0.000
25413.4 <c-61#150:22> Joined cleanup thread
25413.4 <c-61#150:22> Exit destructor
25413.4 <c-61#150:22> Destructing logger
